libfreerdp/core/input.c

```diff
@@ -279,6 +311,8 @@ static void input_send_fastpath_keyboard_pause_event(rdpInput* input)
 		| FASTPATH_INPUT_KBDFLAGS_RELEASE;
 
 	s = fastpath_input_pdu_init_header(rdp->fastpath);
+	if (!s)
+		return FALSE;
```

Comment:
fastpath_input_pdu_init_header returns a stream with 256 byte or FALSE. We only write 8 bytes plus headers here.